scania-digital-design-system/sdds
# fix(link): renamed linkHref to href

**Describe pull-request**  
Renamed linkHref to href.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Link -> Web Components

**Suggested test steps**
